### ISSUE TITLE
GM edit Players Sheet

### ISSUE THREAD

**Issue by graylion-wa** [worldanvil/worldanvil-bug-tracker#19]
**World Anvil Username**: 

Graylion 

**Describe the bug**

The Pencil Icon does not show up on a Players Sheet within the article for the world owner/GM to edit. I can edit the sheet with the url or by going to the character in the campaign.

**Expected behavior**

The pencil icon should show up beside the eye/view icon for a players sheet embedded in an article so the world owner/gm can edit the character.

**Screenshots**

![20210302_165154.jpg](https://user-images.githubusercontent.com/79761754/109720703-6a05f780-7b78-11eb-9124-2338d99cd240.jpg)

Thank you

**Comment by iamromeo** [comment]
There is no context to connect those two.

**Comment by graylion-wa** [comment]
My explanation is most likely poor.

I have attached a character sheet before I hand over to the player (edit button is present).

I have attached the same character sheet after I hand them over to the player (edit button goes away).

I have also attached the image of being able to edit the character sheet in the campaign.

I can also paste the edit sheet link directly into a browser. I may have misunderstood the goal of the pencil as to reduce clicks. Should this be a feature request instead?

Thank

Before
 you![20210304_004146.jpg](https://user-images.githubusercontent.com/79761754/109917452-63b07200-7c83-11eb-9deb-15a3fa1513c4.jpg)

After

![20210304_004408.jpg](https://user-images.githubusercontent.com/79761754/109917480-7034ca80-7c83-11eb-93a7-b3cf3bfdec84.jpg)

Edit

![20210304_004245.jpg](https://user-images.githubusercontent.com/79761754/109917486-74f97e80-7c83-11eb-986f-5d776d36715a.jpg)
